client/modules/_hooks/src/datafiles/usePathDisplayName.ts

```diff
@@ -10,19 +14,20 @@ export function getSystemRootDisplayName(api: string, system: string): string {
       'designsafe.storage.default': 'My Data',
       'designsafe.storage.frontera.work': 'My Data (Work)',
       'designsafe.storage.community': 'Community Data',
-    }[system] ?? 'Data Files'
+    }[system] ?? label
```

Comment:
Can we use this same logic to determine the breadcrumb?
<img width="293" alt="Screenshot 2024-06-03 at 2 45 44 PM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/49e51659-f367-4ab8-9073-5740fec95bf5">

e.g. in this case, instead of 'Data Files', it would say 'Published (NEES)'

Reply:
good point, will do

Reply:
Fixed 

<img width="750" alt="Screenshot 2024-06-03 at 3 06 26 PM" src="https://github.com/DesignSafe-CI/portal/assets/2568355/1ec8a963-39bb-431d-8d78-f59562c01bdf">